### DISCUSSION TITLE
My education pack is approved but still i am not able to redeem benefits.

### DISCUSSION THREAD

**Discussion by mahimtiwari** [community/community#114722] [GitHub Education] [answered]
### Hi everyone,

![Screenshot (352)](https://github.com/community/community/assets/117565856/d9eb9b03-0ba5-4c7b-86c2-521c90476976)
My application is approved but still i am not able to access anything.

**Comment by samus-aran** [comment, marked as answer]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.

**Reply by mahimtiwari** [reply]
thanks

**Reply by mahimtiwari** [reply]
i just forgot to close the q
